site/src/sphinx/server-annotated-service.rst

```diff
@@ -104,6 +104,23 @@ a path and use the HTTP method annotations without a path to map multiple HTTP m
         public HttpResponse hello() { ... }
     }
 
+Every service method assumes that they returns an HTTP response with ``200 Ok`` status. So If you want to
+return another status for a specific method, you can use :api:`@StatusCode` annotation as follows.
+
+.. code-block:: java
+
+    public class MyAnnotatedService {
+
+        @Get("/user/{name}")
+        public HttpResponse getUser(@Param("name") String name) { ... }
```

Comment:
This method could be removed?

Reply:
Added more examples. Could you check them again? Then, please let me know if it   still looks unnecessary. :-)